File: bench/Reference/file_write_read_large.py
with open("bench/data/large_text.txt", "rb") as f:
    data = f.read()
with open("/tmp/py_bench_large_write.txt", "wb") as f:
    f.write(data)
with open("/tmp/py_bench_large_write.txt", "rb") as f:
    data2 = f.read()
print(len(data2))

File: bench/Reference/file_write_read_large.erl
#!/usr/bin/env escript
%% Read 50MB → write → read → print length.

main(_) ->
    {ok, Data} = file:read_file("bench/data/large_text.txt"),
    ok = file:write_file("/tmp/erl_bench_large_write.txt", Data),
    {ok, Data2} = file:read_file("/tmp/erl_bench_large_write.txt"),
    io:format("~p~n", [byte_size(Data2)]).
